src/test/java/nextstep/subway/line/domain/LineTest.java

```diff
@@ -0,0 +1,85 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.station.domain.Station;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class LineTest {
+    private Line line = new Line();
+    Station 양재 = new Station("양재");
+    Station 광교 = new Station("광교");
+    Station 정자 = new Station("정자");
+    Station 강남 = new Station("강남");
+
+    @DisplayName("지하철 구간을 등록한다.")
+    @Test
+    void addLineSection() {
+        line.addSection(양재, 광교, 10);
+
+        List<Station> stations = line.getStations();
+        assertThat(stations).containsExactlyElementsOf(Arrays.asList(양재, 광교));
```

Comment:
`containsExactly` 를 사용하면 리스트로 변환하지 않고 검증할 수 있습니다 :)

```suggestion
        assertThat(stations).containsExactly(양재, 광교);
```